### PR TITLE
Allow resizing right panel

### PR DESCRIPTION
**Pull request by davidmurray**

### CODE REVIEW

**Comment by davidmurray** [comment]
Do not merge yet. Still have to fix some flickering:

https://github.com/chairemobilite/transition/assets/545502/c4f9fcd0-8317-4021-8f4a-a688cc91bc67

**Comment by davidmurray** [comment]
Looks like this now:
https://github.com/chairemobilite/transition/assets/545502/d1d825b3-156d-4518-9258-9ab59bc771c5

We hide the map when resizing to prevent some intense flickering

**Review by kaligrafy — APPROVED** [review]
Wow!

File: packages/chaire-lib-frontend/src/components/dashboard/SplitView.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import React, { createRef, useEffect, useState } from 'react';
```
Comment (tahini) [inline packages/chaire-lib-frontend/src/components/dashboard/SplitView.tsx:1]: Copyright header is missing in this new file, see any other code file for examples

**Review by tahini — COMMENTED** [review]
Nice! My review comes too late though...

**Pull request merged**